programs/uxd/src/state/alloyx_vault_depository.rs

```diff
@@ -0,0 +1,158 @@
+use std::mem::size_of;
+
+use anchor_lang::prelude::*;
+
+use crate::error::UxdError;
+use crate::utils::checked_add;
+use crate::utils::checked_sub;
+
+pub const ALLOYX_VAULT_DEPOSITORY_RESERVED_SPACE: usize = 800;
```

Comment:
Not aligned as it is (5 bool/u8, could be offsetted by this. Not sure how important is the alignement in Solana)

Reply:
Sounds good, will make sure it's a multiple of maybe 64 or 128 in the next PR, not modifying this to not undo your approval for now. Will do next PR 👌